public/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Chords",
+    "short_name": "Chords",
+    "start_url": "/Chords-Web/",
```

Comment:
start URL here should also be `/`, can you check for a way to auto pick this as per repo.